client/assets.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // Assets hold the alice-lg frontend build
-//go:embed build/*
+// go:embed build/*
```

Comment:
this just prevents Go from embedding the files.

The client needs to be built first.
I guess changing the order of running the tests and building the app would fix this.